frontend/src/components/print/print-client/generatePdfMixin.js

```diff
@@ -43,7 +43,7 @@ export const generatePdfMixin = {
         blob,
         slugify(this.config.documentName, {
           locale: this.$store.state.lang.language.substring(0, 2),
-        })
+        }) + '.pdf'
```

Comment:
Because otherwise you can create invalid route urls (material list route). Slugify only has a global instance of the char replacement map. I need to add these characters because otherwise you can't load material lists with an @ in the name. (Which itself is not uncommon, as we use the email address for the list name). If we slugify the documentName (which included `.pdf`) then, the resulting filename was Sola-2023-pdf.